src/test/java/seedu/duke/command/SeeAllExpensesCommandTest.java

```diff
@@ -0,0 +1,54 @@
+package seedu.duke.command;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import seedu.duke.financial.Expense;
+import seedu.duke.financial.FinancialEntry;
+import seedu.duke.financial.FinancialList;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+
+
+public class SeeAllExpensesCommandTest {
```

Comment:
Perhaps adding Javadoc comments for the class would be good?

Reply:
Yeah of course. Working on it.